src/multipoints.jl

```diff
@@ -25,6 +25,8 @@ struct MultiPoint <: AbstractMultiPoint{Point}
     points::Vector{Point}
 end
 
+Base.:(==)(p1::MultiPoint, p2::MultiPoint) = p1.points == p2.points
```

Comment:
Why are all the `==` methods needed?

Reply:
Just for testing (mentioned in header)

It was hard to test read/write round trips without `==` working.